image/gimp_brush.ksy

```diff
@@ -0,0 +1,38 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
+  endian: be
+doc-ref: https://gitlab.gnome.org/GNOME/gimp/-/raw/master/devel-docs/gbr.txt
+seq:
+  - id: header_size
+    type: u4be
+  - id: header
+    type: header
+    size: header_size
+types:
+  header:
+    seq:
+      - id: version
+        type: u4be
+      - id: width
+        type: u4be
+      - id: height
+        type: u4be
+      - id: color_depth
+        type: u4be
```

Comment:
I'd rename this to `bytes_per_pixel`, because it's more explicit and less abstract and vague than `color_depth`. All the more so that the term "color depth" usually refers to the number of **bits per pixel** (not ~bytes~), so abusing the same term here might be actually deceptive and ambiguous. See https://en.wikipedia.org/wiki/Color_depth:

> **Color depth** or **colour depth** (see [spelling differences](https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences#-our,_-or)), also known as [**bit depth**](https://en.wikipedia.org/wiki/Bit_depth_(computer_graphics)), is either the number of bits used to indicate the color of a single pixel

Reply:
@armijnhemel This has not yet been resolved.

Reply:
It is now :-) Sorry for being slow, lots of other tasks to juggle.